test/lib/eslint/BaseEslintEngine.test.ts

```diff
@@ -95,24 +96,6 @@ describe('Tests for BaseEslintEngine', () => {
 
 				expect(results).to.be.empty;
 			});
-
```

Comment:
This is the if condition that was removed from BaseEslintEngine. The caller shouldn't pass rules from another engine to this engine.